### PR TITLE
added test

### PR DESCRIPTION
**Pull request by FiruzKholmatov**
Added some test. Please approve!

### CODE REVIEW

**Review by DaryaDubrovskaya — APPROVED** [review]
Approved

File: src/test/java/school/redrover/GroupUnitedByJava8Test.java [hunk 1]
```diff
@@ -81,4 +81,22 @@ public void testCreateNewUserGlobalQAAsManager() {
         WebElement searchResultsTable = driver.findElement(By.xpath("//tbody/tr[last()]/td[1]"));
         Assert.assertEquals(searchResultsTable.getText(), GLOBALS_QA_STRING_VALUE);
     }
+
+    @Test
+    public void happyPathLogin() {
```
Comment (Gr0zpu) [inline src/test/java/school/redrover/GroupUnitedByJava8Test.java:86]: method name must contain 'Test'

File: src/test/java/school/redrover/GroupUnitedByJava8Test.java [hunk 2]
```diff
@@ -81,4 +81,22 @@ public void testCreateNewUserGlobalQAAsManager() {
...
+
+        WebElement name = getDriver().findElement(By.xpath("//input[@id='user-name']"));
+        name.sendKeys("visual_user");
+
+        WebElement password = getDriver().findElement(By.xpath("//input[@id='password']"));
+        password.sendKeys("secret_sauce");
+
+        getDriver().findElement(By.id("login-button")).click();
+
+        String expectedResult = getDriver().getCurrentUrl();
```
Comment (Gr0zpu) [inline src/test/java/school/redrover/GroupUnitedByJava8Test.java:97]: I thins this is actual result; expected result is "https://www.saucedemo.com/inventory.html"

**Review by Gr0zpu — CHANGES_REQUESTED** [review]

File: src/test/java/school/redrover/GroupUnitedByJava8Test.java (hunk 1 above)
Comment (FiruzKholmatov) [reply src/test/java/school/redrover/GroupUnitedByJava8Test.java:86]: Thanks! Done!

File: src/test/java/school/redrover/GroupUnitedByJava8Test.java (hunk 2 above)
Comment (FiruzKholmatov) [reply src/test/java/school/redrover/GroupUnitedByJava8Test.java:97]: Thanks! Done!

**Pull request merged**